testing/PIMS.Tests.Automation/Classes/Property.cs

```diff
@@ -97,7 +97,7 @@ public class PropertyActivity
     {
         public int PropertyActivityPropsCount { get; set; } = 0;
         public string PropertyActivityType { get; set; } = null!;
-        public string PropertyActivitySubType { get; set; } = null!;
+        public List<string> PropertyActivitySubType { get; set; } = new List<string>();
```

Comment:
nit: naming of this property should indicate that it is a list

```suggestion
        public List<string> PropertyActivitySubTypes { get; set; } = new List<string>();
```